svg/compiler/Makefile

```diff
@@ -77,3 +77,13 @@ clean-js:
 	rm -f ${BINDIR}/js-emitter ${BINDIR}/js-passes* ${BINDIR}/pass-js-emit*
 	rm -f temp.js
 
+junk:
```

Comment:
What does this do?

Reply:
"make junk" is wip (work in progress) for my use - just a way to incrementally build the compiler and see what is failing.  I push wip's, since it makes it easy (for me) to switch between machines.  Branch "center" is mostly a wip, too, as I progress towards accepting ellipses.  Branch "center" was a big-enough change that I thought I should PR it.  Using github for code under development - not yet fully working - is an issue.